python/cudf/cudf/core/frame.py

```diff
@@ -1288,89 +1288,6 @@ def _quantiles(
         result._copy_type_metadata(self)
         return result
 
-    @_cudf_nvtx_annotate
-    def rank(
```

Comment:
This method is moved to `indexed_frame`, since `index.rank` is unsupported in pandas.